### ISSUE TITLE
subdivide abnormal gait

### ISSUE THREAD

**Issue by obophenotype-user** [obophenotype/mammalian-phenotype-ontology#908]
We currently have 474 annotations with notes to abnormal gait. I think we should consider subdividing this term. Some potential children are:
wobbly gait (synonyms unstable?, tottering?, shaky?)
wide gait
shuffling gait
hopping gait
stiff gait (synonym stilted)
uncoordinated gait (synonym clumsy) 
swaying gait
high-stepping (def - bring the legs tight to the body with each step, may involve rear limbs, front limbs or both) (all the examples I came across involved the rear limbs)
bouncy or bouncing gait
tiptoe gait 
hobbling gait
halting gait
staggering gait
waddling gait 

Thanks,
Sue

Reported by: sbello

Original Ticket: [obo/mammalian-phenotype-requests/908](https://sourceforge.net/p/obo/mammalian-phenotype-requests/908)

**Comment by sbello** [comment]
This is now over 1300 annotations

**Comment by sbello** [comment]
Things that seem relatively easy to define

abnormal toe walking (~20 annotations)
def: abnormal gait pattern characterized by the failure of the heel to contact the floor during some or all of the gait
https://www.ncbi.nlm.nih.gov/sites/books/NBK560610/ 
HP:0030051 
definition is currently pulled from HPO, I'm uncertain if 'onset' is correct for mouse gait, looking into annotations, most annotations just say the mice walk on their toes so I've removed 'onset' and added 'some or all' 
calling this abnormal toe walking as some mammals normally walk on their toes.

**Comment by sbello** [comment]
add abnormal gait width
related synonym: abnormal stance width
def: any anomaly in the distance between the right and left feet when moving
child terms: increased/decreased gait width
related synonym wide/narrow stance
synonym: wider gait/narrow gait

**Comment by sbello** [comment]
abnormal hopping gait - anomalous gait with synchronized lifting of the forelimbs followed by out of phase synchronized lifting of the hindlimbs or by synchronized lifting of just the hindlimbs in a mammal which does not normally display this type of gait at the movement speed examined - PMID:24623763, PMID:20014422
synonym - (narrow) quadrupedal hopping gait, rabbit-like hopping, hopping kangaroo gait

**Comment by sbello** [comment]
Review of 'waddle/waddling'
J:266 uses duck-like as synonym for waddle - described as legs held wider apart and as ataxic
J:93329 - waddle = "Step width increases in relation to step length, and four tracks of prints become visible instead of two."
J:134789 - shorter, “waddling” steps - image seems to show 4 tracks rather than 2
PMID:9305844 - waddling - characterized by a swinging of the rear body
PMID:9731539 - marked ataxia characterized by a waddling, uncoordinated gait 
most papers don't bother to say more than waddle or waddling - unlikely to easily distinguish this from wobbly or unstable. So skip this.

review of shuffling gait
J:148340 - nonuniform alternating left-right step pattern; methods definition "shuffle gait in which all alternate steps fell exactly beside the preceding opposite steps "
J:117465 - shuffling gait, characterized by a reduced gait width and a consistent failure to finish the tandem walk 
most of the rest of the references just say shuffling - with no clear, consistent definition skip this

review of hindlimb dragging
J:84392 - unusual gait involving dragging of the hindlimbs
J:88460 - shuffle gait with dragging hind limbs
J:76203 - abnormal gait characterized by decreased flexibility of knee and ankle joints (i.e., dragging leg) only in double-deficient mice (see Fig. 1A). The dragging was transient and could occur on and off on the right or left side
J:93411 - dragging of the hind limbs
J:47999 - dragging of hindquarters during forward locomotion
proposed term - abnormal hindlimb dragging
def: abnormal failure to lift all or part of the hindlimbs off the ground during locomotion

**Comment by anna-anagnostop** [comment]
review of halting gait
J:6580 - slow halting gait, with all feet flat against the surface
J:311491 - mice develop a halting gait by 4 months of age (paper does no elaborate further)
J:70736 - wobbling and halting gait (paper says "mice show wobbling and halting behaviors during walking but do not exhibit an overt ataxic gait")
J:9279 - [htr](https://www.informatics.jax.org/allele/MGI:1856908)/[htr](https://www.informatics.jax.org/allele/MGI:1856908) hesitator mice have a halting gait
J:282431 - mice exhibit an aberrant halting gait and frequently lose their balance
skip this?

**Comment by sbello** [comment]
Yes, I would skip halting. I'm not including things unless I can find consistent usage across multiple papers with more than 2 giving a definition in the paper.

bouncy - only has 4 mentions and none define, some use in relation to unsteady or ataxic

uncoordinated seems to be used to indicate clumsy, ataxic, wobbly etc, none of these are well defined 

J:223127 - foot base angle "foot-base-angle at toe-off positions of the hind-paws was measured using single video frames from recordings of
beam walking mice" looking at the image this appears to be the angle of the foot as it lifts off the ground relative to the ground - this does NOT appear to be the same as step angle or paw angle
J:225345 - step angle - from DigiGait Imaging and Analysis system - 

Looking for explanation of the DigiGait parameters
J:115355 - earliest paper has 3 parameters
- Stride time is the time between two initial paw contacts with the treadmill for the same paw. 
- Stance time is the portion of the stride time that the paw is in contact with the treadmill,
- Swing time is the portion of the stride time that the paw is not in contact with the treadmill surface. 
no mention of angle
PMID:15608600
- Paw angles were automatically calculated at peak stance between the long axes of the elliptical paw images and the centerline of the mouse Fig 6 has a good image if this - this looks much like the in/out toe from HPO, note that the forelimb and hindlimb angles are different
- Forelimb and hind limb stance widths were calculated as the perpendicular distance between the major axes of the left and right paw images during peak stance
-  stride interval, which includes the stance duration when the paw is in contact with the walking surface, plus the swing duration when the paw is not in contact with the walking surface (Clarke and Still, 1999, 2001; Leblond et al., 2003; Yu et al., 2001). Stance duration was further subdivided into either braking duration (increasing paw contact area over time) or propulsion duration (decreasing paw contact area over time). Full stance was determined as the time point at which the paw contact area was maximum
- Stride frequency was calculated by counting the number of gait signals over time

PMID:16042805
-  stride duration (stride time), which includes the stance duration when the paw of a limb is in contact with the walking surface, plus the swing duration when the paw of the same limb is not in contact with the walking surface. Stance duration was further subdivided into braking duration (increasing paw contact area over time) and propulsion duration (decreasing paw contact area over time) 
- Stride frequency was calculated by counting the number of gait signals over time
- Stride length was calculated from the equation: speed = stride frequency × stride length
- stance widths and paw placement angles at full stance, ellipses were fitted to the paws, and the centers, vertices, and major axes of the ellipses were determined - Fig 1 has good image for angle
- Forelimb and hind limb stance widths were calculated as the perpendicular distance between the major axes of the left and right paw images during peak stance
- stride-to-stride variability (gait variability) for stride length, stride time, and stance width were determined as the standard deviation and the coefficient of variation

**Comment by sbello** [comment]
add
abnormal step angle
any anomaly in the angle between the long axis of the foot and the centerline of the mammal 
PMID:15608600
synonym - abnormal paw angle
child terms 
- increased/decreased
- hindlimb/forelimb with related increased/decreased

**Comment by sbello** [comment]
J:94884 - feet pointed outward (duck walk) possibly duck walk is a synonym for increased step angle?
J:67077 - froglike gait with hindlimbs splayed outward during locomotion - possibly related to foot base angle
lots or references to splayed hindlimbs but this seems less gait and more how the mouse stands
